decidim-bulletin_board-app/app/javascript/packs/sandbox/vote.js

```diff
@@ -79,12 +84,21 @@ $(async () => {
       $encryptVote.prop("disabled", true);
       $castVote.show();
       $auditVote.show();
+      $storeVote.show();
     },
     onBindAuditBallotButton(onEventTriggered) {
       $auditVote.on("click", onEventTriggered);
     },
     onBindCastBallotButton(onEventTriggered) {
+      $castVote.on("click", () => {
+        setPerformedAction(CAST_ACTION);
+      });
       $castVote.on("click", onEventTriggered);
+
+      $storeVote.on("click", () => {
+        setPerformedAction(STORE_ACTION);
+      });
+      $storeVote.on("click", onEventTriggered);
```

Comment:
Minor refactors because I don't like binding so many events:
```js
$castVote.on("click", event => {
  setPerformedAction(CAST_ACTION);
  onEventTriggered(event);
})

$storeVote.on("click", event => {
  setPerformedAction(STORE_ACTION);
  onEventTriggered(event);
});
```